### ISSUE TITLE
Getting content to properly fit printed page

### ISSUE THREAD

**Issue by logidelic** [katzer/cordova-plugin-printer#114]
Nice plugin. Unfortunately I'm having trouble controlling the output, to fit correctly for the target paper size. Are there known issues?

For example, if my target paper (a label in this case) is 3.5in x 1.1in and I specify these dimensions for the DIV in CSS, it is printed much smaller than that. I have tried a number of different alternative layouts for my document, but none fit correctly when printing via this plugin.

On the other hand, all work correctly, when printing via a web browser.

Any ideas about the possible differences here? I assume that the plugin be able to render things the same way as Safari?

**Comment by katzer** [comment]
@logidelic iOS or Android?

**Comment by logidelic** [comment]
@katzer, this is iOS; sorry for not mentioning.

After much trial-and-error, I found that if scaled up by a fudge factor, I was able to get things to print correctly. For example, instead of specifying a width of 3.5in in CSS (the actual width of the paper), I specified 4.9in.

For all I know, this may be a bug in iOS Safari or the particular printer's (Brother QL-720NW) AirPrint implementation so I can't fairly point the finger at the plugin...
